File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
           string y = to_string(x);
        for (int i=0;i<y.length();i++)
        {
            if (y[i]==y[y.length()-1-i])
                continue;
            else 
                return false;
        }
        
          return true;
        
    }
};

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x<0)
            return false;
    String number = new String();
        number=Integer.toString(x);
        Integer length=number.length();
        for (int i=0;i<length/2;i++)
        {
            if (number.charAt(i)!=number.charAt(length-1-i))
                return false;
        }
        return true;
    }
}